test/cookbooks/resources_test/metadata.rb

```diff
@@ -0,0 +1,9 @@
+name 'resources_test'
+maintainer 'Oregon State University'
+maintainer_email 'systems@osuosl.org'
+license 'Apache-2.0'
+description 'Installs/Configures multi_node_test'
+version '0.1.0'
+chef_version '>= 13.0'
```

Comment:
For `metadata.rb` we like to format it like the following:
```suggestion
name                'resources_test'
maintainer          'Oregon State University'
maintainer_email    'systems@osuosl.org'
license             'Apache-2.0'
description         'Installs/Configures resources_test'
version             '0.1.0'
```